api/libproject/devstart.rb

```diff
@@ -802,6 +802,16 @@ def deploy(cmd_name, args)
     lambda {|opts, v| opts.quiet = true},
     "Don't display a confirmation prompt when deploying"
   )
+  op.add_option(
+    "--deploy-cron",
+    lambda {|opts, v| opts.deploy_cron = true},
+    "Also deploy cronjobs"
+  )
+  op.add_option(
```

Comment:
We don't actually read --no-deploy-cron below... I think we can get rid of it and just go with the absence of deploy-cron?

Reply:
This would allow one to override on way or the other, e.g. deploy API without cron in the event that were slower or there's some issue with it. With the flag, this felt asymmetric otherwise since deploy-api always sets `--deploy-cron`.

Alternatively I could just change this to be a normal parameter and not use this options stuff.

Reply:
On further thought, I don't see any reason to make this flag, as it would just result in failures for `deploy-public-api`. Changed to a param.